GliaWidgets/View/Call/CallView.swift

```diff
@@ -7,8 +7,40 @@ class CallView: EngagementView {
         case upgrading
     }
 
+    var callDuration: String? {
+        didSet {
+            guard !isVisitrOnHold else { return }
+
+            secondLabel.text = callDuration
+            connectView.statusView.setSecondText(callDuration, animated: false)
+        }
+    }
+
+    var isVisitrOnHold: Bool = false {
+        didSet {
+            connectView.operatorView.isVisitorOnHold = isVisitrOnHold
+
+            if isVisitrOnHold {
+                secondLabel.text = style.onHoldTopText
+                connectView.statusView.setSecondText(style.onHoldTopText, animated: false)
+            }
```

Comment:
What happens for `else` case? Does `secondLabel.text` and  `connectView.statusView.setSecondText` still has  text assigned?

Reply:
added else clause. It wasn't strictly needed due to the duration text would then be shown, but for claritys sake I added the duration text on the else